test-infra/terraform/aws/user-data.sh

```
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+GH_OWNER="${gh_owner}"
```

Shellcheck (0.11.0)

[warning] gh_owner is referenced but not assigned (did you mean 'GH_OWNER'?).

(SC2154)